ci/storage/s3.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package storage
+
+import (
+	log "github.com/cihub/seelog"
+	"github.com/magefile/mage/sh"
+	"github.com/mysteriumnetwork/node/ci/env"
+	"github.com/mysteriumnetwork/node/logconfig"
+	"github.com/pkg/errors"
+)
+
+// MakeBucket creates a bucket in s3 for the build (env.BuildNumber)
+func MakeBucket() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	return env.IfRelease(func() error {
+		url, err := bucketUrlForBuild()
+		if err != nil {
+			return err
+		}
+		return sh.RunV("bin/s3", "mb", url)
```

Comment:
There should be some S3 library for golang, we will need to replace bash scripts by it later.

Reply:
Absolutely, this is just work in progress of porting all build scripts.